### PR TITLE
flat-plat: init at 3.20.20160404

### PR DESCRIPTION
**Pull request by Mounium**
###### Things done
- [ ] Tested using sandboxing
  ([nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS,
    or option `build-use-chroot` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by Mounium** [comment]
@joachifm I had to close the previous one, but here it is again with your suggested changes.

File: pkgs/misc/themes/flat-plat/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, gnome_themes_standard }:
+
+stdenv.mkDerivation rec {
+  name = "flat-plat-gtk-theme-${version}";
+  version = "3.20.20160404";
+
+  src = fetchurl {
```
Comment (rasendubi) [inline pkgs/misc/themes/flat-plat/default.nix]: `fetchFromGitHub` might be better.

File: pkgs/misc/themes/flat-plat/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  installPhase = ''
+    mkdir -p $out/share/themes
+    cp -r ../Flat-Plat $out/share/themes
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A Material Design-like flat theme for GTK3, GTK2, and GNOME Shell";
+    homepage = https://github.com/nana-4/Flat-Plat;
+    licence = licenses.gpl2;
+    maintainers = [ mounium ];
```
Comment (rasendubi) [inline pkgs/misc/themes/flat-plat/default.nix]: `maintainers.mounium`

File: pkgs/misc/themes/flat-plat/default.nix (hunk 1 above)
Comment (joachifm) [inline pkgs/misc/themes/flat-plat/default.nix]: If this is a tarball uploaded by the project, I *think* github will leave it alone and serve it as-is, so the hash *should* be stable (compared to their auto-generated tarballs).

File: pkgs/misc/themes/flat-plat/default.nix [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+stdenv.mkDerivation rec {
+  name = "flat-plat-gtk-theme-${version}";
+  version = "3.20.20160404";
+
+  src = fetchurl {
+    url = "https://github.com/nana-4/Flat-Plat/releases/download/${version}/Flat-Plat-${version}.tar.gz";
+    sha256 = "089qyn1sqilyry4bw4l8v3yi55aw9pakbkp59z6r9y23m4j3mdfb";
+  };
+
+  buildInputs = [ gnome_themes_standard ];
```
Comment (rasendubi) [inline pkgs/misc/themes/flat-plat/default.nix]: How `gnome_themes_standard` is used? I mean, build phase is disabled, there is no configure script and install phase doesn't use that either. Is that really needed?

**Pull request merged**